README.md

```diff
@@ -243,11 +243,11 @@ below define how this translation works.  If there is no map for your policy you
             },
 ```
 
-The key `Accounts: Rename administrator account ` in the first hash is what the user will define as the name in the resource name. 
+The key `Accounts: Rename administrator account ` in the first hash is what the user will define as the name in the resource name.
 Instead of remembering the policy name, the description will help us remember what the policy is for.  When defining new policy
-maps you will need to define the key, name, policy_type, and optionally, data_type or reg_type.  
+maps you will need to define the key, name, `policy_type`, and optionally, `data_type` or `reg_type`.
 
-Currently for data_type there is only `:quoted_string`.  However, for reg_type(integer value) there are many values which are listed below:
+Currently for `data_type` there is only `:quoted_string`.  However, for `reg_type` (integer value) there are many values which are listed below:
```

Comment:
"Currently supported values for 'data_type'  are ':quoted_string' and ':principal'.  Use ':principal' when defining a policy that takes users or groups as the value.  For 'reg_type', the following values are supported:"